docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -486,6 +486,42 @@ The following command operators are supported:
   UDP
     Not implemented ("-").
 
+%RESPONSE_FLAGS_FULL%
```

Comment:
I think I'd prefer renaming this to `RESPONSE_FLAGS_LONG`

Reply:
done